my_compassion/templates/my2_assets.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!--
+    These templates inherit from Odoo's "website" templates and adds custom behaviour to it.
+-->
+<odoo>
+    <!-- Make all pages load global.css file -->
+    <template
+            id="my_compassion_assets"
+            name="My Compassion 2.0 assets"
+            inherit_id="website.assets_frontend"
+    >
+        <!-- Loads files at the end of page -->
+        <xpath expr="//script[last()]" position="after">
+            <link
+                    href="/my_compassion/static/src/css/global.css"
+                    rel="stylesheet"
+                    type="text/css"
+            />
+        </xpath>
+    </template>
+    
+    <!-- Add a meta instruction in all page's header -->
+    <template id="my_compassion_meta"
+              name="Add custom meta"
+              inherit_id="website.layout"
+    >
+        <!-- Adds the mobile first optimization in the head of every page -->
+        <xpath expr="//head/meta[last()]" position="after">
+            <meta name="viewport" content="width=device-width, initial-scale=1.0"/>
+        </xpath>
+    </template>
+</odoo>
```

Comment:
Is this compatible with our other websites ? It shouldn't compromise TOGETHER,Muskathlon,... layouts.